docs/content/getting-started/setup-openfga/configuring-openfga.mdx

```diff
@@ -0,0 +1,334 @@
+5. Configure the HTTP TLS key location: 
+`export OPENFGA_HTTP_TLS_KEY=/Users/myuser/key/server.key`
+
+</TabItem>
+
+</Tabs>
+
+To learn how to run in in Docker, check our [Docker documentation](./docker-setup.mdx#oidc-authentication).
+
+### Profiler (pprof)
+:::caution Warning
+Continuous profiling can be used in production deployments, but we recommend disabling it unless it is needed to troubleshoot specific performance or memory problems.
+:::
+
+Profiling through [`pprof`](https://github.com/google/pprof/blob/main/doc/README.md) can be enabled on the OpenFGA server by providing the `--profiler-enabled` flag. For example:
+
+```sh
+openfga run --profiler-enabled
+```
+
+If you need to serve the profiler on a different port than the default `3001`, you can do so by specifying the `--profiler-addr` flag. For example:
+
+```sh
+openfga/openfga run --profiler-enabled --profiler-addr :3002
+```
+
+If you want to run it in docker:
+```sh
+docker run -p 8080:8080 -p 8081:8081 -p 3000:3000 -p 3002:3002 openfga/openfga run --profiler-enabled --profiler-addr :3002
+```
+
+## Health Check
+
+OpenFGA is configured with an HTTP health check endpoint `/healthz` and a gRPC health check `grpc.health.v1.Health/Check`, which is wired to datastore testing. Possible response values are
+- UNKNOWN
+- SERVING
+- NOT_SERVING
+- SERVICE_UNKNOWN
+
+<Tabs groupId={"healthcheck"}>
+<TabItem value={"health-curl"} label={"cURL"}>
+
+```shell
+curl -X GET $FGA_API_HOST/healthz
+
+# {"status":"SERVING"}
+```
+
+</TabItem>
+
+<TabItem value={"health-grpc"} label={"gRPC"}>
+
+```shell
+# See https://github.com/fullstorydev/grpcurl#installation
+grpcurl -plaintext $FGA_API_HOST grpc.health.v1.Health/Check
+
+# {"status":"SERVING"}
+```
+</TabItem>
+</Tabs>
+
+## Experimental Features
+Various releases of OpenFGA may have experimental features that can be enabled by providing the `--experimentals` flag or the `experimentals` config.
+
+```
+openfga run --experimentals="feature1, feature2"
+```
+or if you're using environment variables,
+```
+openfga -e OPENFGA_EXPERIMENTALS="feature1, feature2" run
+```
+
+The following table enumerates the experimental flags, a description of what they do, and the versions of OpenFGA the flag is supported in:
+
+| Name              | Description                                                        | OpenFGA Version       | Status                     |
+|-------------------|--------------------------------------------------------------------|-----------------------|----------------------------|
+| otel-metrics      | Enables support for exposing OpenFGA metrics through OpenTelemetry | `0.3.2 <= v < v0.3.5` | became default in `v0.3.5` |
+| list-objects      | Enables ListObjects API                                            | `0.2.0 <= v < v0.3.3` | became default in `v0.3.3` |
+| check-query-cache | Enables caching of check subproblem result                         | `1.3.1 <= v `         | currently experimental     |
```

Comment:
`check-query-cache` experimental flag was removed in v1.3.6

Reply:
Do we want to mention that conditional relationship tuples was experimental from 1.3.8 to 1.4.0?

Reply:
not blocking for changes - but we may want to remove the whole status section all together as per https://github.com/openfga/openfga.dev/pull/547/files#r1385058120

Reply:
I added the 'enable-conditions' flag, and I removed the 'status' section